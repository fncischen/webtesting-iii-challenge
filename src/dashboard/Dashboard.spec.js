import React from 'react';
import renderer from 'react-test-renderer';

import Display from "../display/display.js"
import Controls from "../controls/controls.js"


describe("Dashboard", () => { 

it('Display is rendering', () => {
  const tree = renderer
    .create(<Display/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('Controls is rendering', () => {
    const tree = renderer
      .create(<Controls/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

});