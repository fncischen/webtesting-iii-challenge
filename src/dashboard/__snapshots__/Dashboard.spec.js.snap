// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Controls is rendering 1`] = `
<div
  className="controls panel"
>
  <button
    className="toggle-btn"
    disabled={true}
  >
    Lock Gate
  </button>
  <button
    className="toggle-btn"
  >
    Close Gate
  </button>
</div>
`;

exports[`Dashboard Display is rendering 1`] = `
<div
  className="display panel"
>
  <div
    className="led green-led"
  >
    Unlocked
  </div>
  <div
    className="led green-led"
  >
    Open
  </div>
</div>
`;
